daprdocs/content/en/concepts/dapr-services/placement.md

```diff
@@ -14,3 +14,59 @@ The placement service Docker container is started automatically as part of [`dap
 ## Kubernetes mode
 
 The placement service is deployed as part of `dapr init -k`, or via the Dapr Helm charts. For more information on running Dapr on Kubernetes, visit the [Kubernetes hosting page]({{< ref kubernetes >}}).
+

```

Comment:
Given this is a new endpoint on the sidecar, this whole topic need to be move to a new topic under observability here https://docs.dapr.io/developing-applications/building-blocks/observability/ similar to health endpoint. This should then be referenced from here https://docs.dapr.io/concepts/dapr-services/sidecar/ with the diagram updated (we docs maintainers can update the diagram) 

I would still add a reference to the article under https://docs.dapr.io/concepts/dapr-services/placement/ service

Reply:
In fact, this is a new endpoint on the placement service, used to get the actors' info registered to placement.

Reply:
Ok. I read that but the healthz port reference threw me, since this is on the Dapr sidecar. So I am still not clear here. If I  do not have any dapr instances running and I simply do "dapr init" on my local machine "curl localhost:8080/placement/state"  this will always return placement information?

And when placement service is deployed to Kubernetes, is there a consistent endpoint I can call then? Can I set up port forwarding, or how would you show an example of this for Kubernetes?

I also suggest adding a diagram into the https://docs.dapr.io/concepts/dapr-services/placement/ overview showing this API, which we can help with

Reply:
> If I do not have any dapr instances running and I simply do "dapr init" on my local machine "curl localhost:8080/placement/state" this will always return placement information?

No, "dapr init" runs placement in docker by running the command `docker run --name dapr_placement --restart always -d --entrypoint ./placement -p 50005:50005 docker.io/daprio/dapr:1.11.2`. The healthz server port `8080` is not exposed. If users run placement binary directly on local machine by `./placement --metadata-enabled`,  this curl will always return placement information. Or I can raise a pr on dapr cli repo 

> And when placement service is deployed to Kubernetes, is there a consistent endpoint I can call then? Can I set up port forwarding, or how would you show an example of this for Kubernetes?

For Kubernetes, this can get after set port forwarding, I'll add an example for this.

> I also suggest adding a diagram into the https://docs.dapr.io/concepts/dapr-services/placement/ overview showing this API, which we can help with

Can you please give an example of the diagram? The current content is referenced to how dapr introduces its metadata api.https://docs.dapr.io/reference/api/metadata_api/

Reply:
@msfussell bump ^^